tasks/post_copy_request_to_queue/test/unit_tests/test_post_copy_request_to_queue.py

```diff
@@ -56,13 +56,11 @@ def setUpQueues(self):
 
     @patch("post_copy_request_to_queue.task")
     def test_handler_happy_path(self, mock_task: MagicMock):
-        key_path = uuid.uuid4().__str__()
-        bucket_name = uuid.uuid4().__str__()
-        event = {
-            "Records": [
-                {"s3": {"object": {"key": key_path}, "bucket": {"name": bucket_name}}}
-            ]
-        }
+        key_path = "test.jpg"
+        bucket_name = "test-bucket"
```

Comment:
I recommend against removing randomized test properties, as I've seen cases where something was being copy/pasted between setup and testing, and therefore invalidated the test.
If you want values that are more debug-friendly, use something like `f"{uuid.uuid4()}.ext"` or `f"{uuid.uuid4()}-bucket"`

Reply:
```
body_json = json.dumps({
    "Records": [
        {
            "s3": {
                "bucket": {
                    "name": bucket_name
                },
                "object": {
                    "key": key_path
                }
            }
        }
    ]
})
event = {"Records": [{"body": body_json}]}
```

Reply:
thanks looks good